db/config.cc

```diff
@@ -949,7 +949,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , shutdown_announce_in_ms(this, "shutdown_announce_in_ms", value_status::Used, 2 * 1000, "Time a node waits after sending gossip shutdown message in milliseconds. Same as -Dcassandra.shutdown_announce_in_ms in cassandra.")
     , developer_mode(this, "developer_mode", value_status::Used, DEVELOPER_MODE_DEFAULT, "Relax environment checks. Setting to true can reduce performance and reliability significantly.")
     , skip_wait_for_gossip_to_settle(this, "skip_wait_for_gossip_to_settle", value_status::Used, -1, "An integer to configure the wait for gossip to settle. -1: wait normally, 0: do not wait at all, n: wait for at most n polls. Same as -Dcassandra.skip_wait_for_gossip_to_settle in cassandra.")
-    , endpoint_expiration_time(this, "endpoint_expiration_time", value_status::Used, std::chrono::seconds(24 * 60 * 60 * 3), "Amount of time (in seconds) an unavailable node should be kept in gossip information.")
+    , endpoint_expiration_time(this, "endpoint_expiration_time", value_status::Used, 24 * 60 * 60 * 3, "Amount of time (in seconds) an unavailable node should be kept in gossip information.")
```

Comment:
I don't understand this change. The commit log should explain why this is done, it makes no sense to me.